### PR TITLE
Fix repos

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: src/clients/migration_proposals.rb [hunk 1]
```diff
@@ -1,3 +1,23 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) 2015 SUSE GmbH, All Rights Reserved.
```
Comment (lslezak) [inline src/clients/migration_proposals.rb]: Should it be `SUSE LLC`? I'm not sure, but in `registration` and `journal` there is `SUSE LLC`.

Who is actually the correct copyright holder now?

Comment (jreidinger) [inline src/clients/migration_proposals.rb]: hmm, I use it same as in spec file which is done automatically so I expect it have the newest copyright holder as it also refresh date: https://build.suse.de/package/view_file/SUSE:SLE-12:GA/MozillaFirefox/MozillaFirefox.spec?expand=1

Comment (lslezak) [inline src/clients/migration_proposals.rb]: Ok, it's just confusing to see several different values there (SUSE Gmbh, SUSE LLC, Novell)...

**Comment by lslezak** [comment]
LGTM

**Pull request merged**